### DISCUSSION TITLE
query timeout and has error

### DISCUSSION THREAD

**Discussion by RaphaelL2e** [milvus-io/milvus#16078] [Q&A and General discussion]
curl 'http://127.0.0.1:3000/api/v1/collections/m_7344ac970a85be10c8e0083f0dfbc8cc/search' \
  -H 'Connection: keep-alive' \
  -H 'Accept: application/json, text/plain, */*' \
  -H 'milvus-address: 10.10.20.105:39530' \
  -H 'User-Agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) attu/2.0.1 Chrome/96.0.4664.55 Electron/16.0.2 Safari/537.36' \
  -H 'Content-Type: application/json' \
  -H 'Sec-Fetch-Site: cross-site' \
  -H 'Sec-Fetch-Mode: cors' \
  -H 'Sec-Fetch-Dest: empty' \
  -H 'Accept-Language: en-US' \
  --data-raw '{"output_fields":["pk"],"expr":"","search_params":{"params":"{\"nprobe\":1,\"round_decimal\":-1}","anns_field":"embeddings","topk":100,"metric_type":"L2","round_decimal":-1},"vectors":[[0.08574746549129486,-0.010763579048216343,-0.019088687375187874,-0.07065729796886444,-0.007599550299346447,-0.09666039049625397,-0.03551119193434715,-0.02770850993692875,0.01820271834731102,0.048722587525844574,-0.13175193965435028,0.0624188669025898,-0.026642804965376854,0.08729186654090881,-0.08836686611175537,0.07594337314367294,0.0651620402932167,-0.028375213965773582,-0.07874894887208939,-0.10185538232326508,-0.09000066667795181,0.06397126615047455,0.0423751063644886,0.11740142107009888,0.09314914047718048,0.15773001313209534,0.05335909500718117,0.12113671749830246,-0.0852922648191452,0.10057462006807327,0.10298933833837509,-0.021239977329969406,-0.09465672820806503,0.15024614334106445,-0.14438964426517487,-0.03537644073367119,0.04915274679660797,0.055555153638124466,0.09997126460075378,0.018353857100009918,0.0010926115792244673,-0.07446124404668808,0.2011161893606186,0.012715315446257591,0.029665933921933174,-0.10850286483764648,-0.06554827839136124,0.05880649760365486,0.01885112375020981,-0.08207717537879944,0.06296756118535995,0.026668034493923187,-0.038780953735113144,0.12350187450647354,-0.07448960095643997,0.05778462812304497,0.04223133623600006,-0.016148561611771584,0.1661524623632431,-0.17436039447784424,-0.09640196710824966,-0.012979512102901936,0.06384316086769104,0.0868859514594078,-0.010554204694926739,0.12864544987678528,0.14989814162254333,-0.17677104473114014,-0.09837818145751953,0.013576394878327847,-0.05140257626771927,0.07985565066337585,-0.1718091070652008,0.01604646071791649,0.06363478302955627,0.05166052281856537,0.004215289372950792,-0.010954133234918118,-0.06797057390213013,0.02593878097832203,-0.04926039278507233,-0.06474940478801727,0.11506599932909012,0.005191086791455746,0.08288737386465073,0.0686161071062088,0.0010874634608626366,-0.04136272147297859,-0.010823243297636509,0.1051979660987854,0.040168292820453644,-0.07982505112886429,0.011887701228260994,0.11153256148099899,0.07135629653930664,-0.09117675572633743,-0.08329503238201141,-0.019547808915376663,-0.08697652816772461,-0.10640707612037659,-0.057740576565265656,-0.004671224858611822,0.21960081160068512,-0.12639640271663666,-0.04383334517478943,-0.03707696124911308,-0.1284492462873459,0.04205220565199852,0.21596881747245789,-0.15747971832752228,-0.08534353226423264,-0.05481830611824989,0.09602872282266617,-0.02406204119324684,-0.07055964320898056,-0.03749825432896614,-0.13549625873565674,-0.06969384104013443,-0.12728828191757202,-0.07754448801279068,0.08364927768707275,0.03838273137807846,-0.07473525404930115,-0.027835553511977196,-0.10813120007514954,-0.1097349226474762,0.19903840124607086,0.0008711767150089145]],"vector_type":101}' \
  --compressed

![83fbb6f1ce8593e6d3193a5f9b217b0](https://user-images.githubusercontent.com/37803042/158573949-59c0b9c8-a8c1-4811-b6bd-c912aa0e42bf.jpg)

Search failed, C Runtime Exception: [UnexpectedError] Assert \"adapter->CheckSearch(conf, index_type, field_indexing->indexing_->index_mode())\" at /go/src/github.com/milvus-io/milvus/internal/core/src/query/SearchOnSealed.cpp:91\n => [SearchOnSealed]Search params check failed\n"]
Assert "adapter->CheckSearch(conf, index_type, field_indexing->indexing_->index_mode())" at /go/src/github.com/milvus-io/milvus/internal/core/src/query/SearchOnSealed.cpp:91
 => [SearchOnSealed]Search params check failed

**Comment by yhmo** [comment]
The "Search params check failed" means the search parameters are illegal. Is the index type IVF?
